SWLOR.Game.Server/Feature/LootTableDefinition/DathomirLootTableDefinition.cs

```diff
@@ -495,6 +496,32 @@ private void JungleBug()
                 .AddItem("map_68", 2, 1, true)
                 .AddItem("map_69", 2, 1, true)
                 .AddItem("emerald", 1, 1, true);
+
+
+        }
+
+        private void GapingSpider()
+        {
+            _builder.Create("DATHOMIR_GAPING_SPIDER")
+                .AddItem("lth_high", 10)
+                .AddItem("lth_imperfect", 5)
+                .AddItem("spider_guts", 10)
+                .AddItem("spider_leg", 5);
+
+            _builder.Create("DATHOMIR_GAPING_SPIDER")
```

Comment:
This second one is replacing your first loot table. I think you meant for this to be the rares, but the ID is the same. Please fix.